src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1336,7 +1353,7 @@
 examples:
   - name: List all CORS rules for a storage account. (autogenerated)
     text: |
-        az storage cors list --account-key 00000000 --account-name mystorageaccount
+        az storage cors list --account-name MyAccount
```

Comment:
reject this one.
storage account should be lowercase

Reply:
```suggestion
        az storage cors list --account-key 00000000 --account-name mystorageaccount
```